NOTICE

```diff
@@ -0,0 +1,2 @@
+Copyright 2020-2023, Frank Zhao and the Hypercrx contributors
```

Comment:
Do we need to use my name here? Can we use X-lab instead of Frank Zhao?

Reply:
Your suggestion should work, I suppose.